Automattic/pocket-casts-android
# Bookmarks - Add search on podcast page

## Description

This adds bookmark search on the podcast page.

## Testing Instructions
1. Launch the app
2. Play an episode and add a few bookmarks
3. Go to the corresponding `Podcast Details` page
5. Tap on bookmarks tab
6. Search for bookmark title
7. ✅ Notice that the bookmark is displayed
8. Search for bookmark's episode title
9. ✅ Notice that the bookmark for that episode is displayed

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/1405144/b4a4b9e6-9aee-40fc-b1e3-ea9ee42dd764

## Checklist
 
#### I have tested any UI changes...
